docs/src/model_implementation.md

```diff
@@ -312,5 +314,33 @@ fig
 
 We can see in this that some phytoplankton sink to the bottom, and are both remineralized back into nutrients and stored in the sediment.
 
+### Running on a GPU
+
+In order to run a BGC model on a GPU, the BGC model must first be `adapted`. After the definition of the BGC `struct`, we need to write:
```

Comment:
It might be good to add a link to [this](https://clima.github.io/OceananigansDocumentation/stable/simulation_tips/#Arrays-in-GPUs-are-usually-different-from-arrays-in-CPUs) explanation of what is going on here